File: backend/GamesMarket.Domain/Entities/Publisher.cs
using GamesMarket.Domain.Enums;

namespace GamesMarket.Domain.Entities
{
    public class Publisher : Entity
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string Document { get; set; }
        public TypePerson TypePerson { get; set; }
        public DateOnly FoundationDate { get; set; }

        public Address Address { get; set; }
        public IEnumerable<Game> Games { get; set; }
    }
}

File: frontend/src/app/modules/publisher/models/publisher.ts
import { Address } from './address';
import { Game } from 'src/app/modules/games/models/games.model';

export class Publisher {
    id: string;
    name: string;
    email: string;
    document: string;
    typePerson: string;
    foundationDate: string;
    address: Address;
    games: Game[];
}

